.idea/codeStyles/Project.xml

```diff
@@ -46,7 +46,6 @@
     </JavaCodeStyleSettings>
     <XML>
       <option name="XML_ALIGN_ATTRIBUTES" value="false" />
-      <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
```

Comment:
Auto-changed by IntelliJ. 
Can remove from the PR.

Reply:
probably is fine, what version are you running?

Reply:
```
IntelliJ IDEA 2021.2 (Ultimate Edition)
Build #IU-212.4746.92, built on July 27, 2021
```

Reply:
yeah fine, i'm on the previous version 2021.1 but i'm sure no issue with removing